src/components/article/ViewArticle.jsx

```diff
@@ -16,6 +17,12 @@ class ViewArticle extends React.Component {
   componentDidMount = async () => {
     const { match: { params }, getArticle: viewArticle } = this.props;
     await viewArticle(params.slug);
+    const { message, history } = this.props;
+    if (typeof message !== 'string') {
+      setTimeout(() => {
```

Comment:
@marcdomain Nice work. But I think making use of setTimeout here will add to the time it going to take for the page to load when the user is directed to it. What do you think?